doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -286,7 +286,7 @@ Float64
 Half-precision floating-point numbers are also supported (`Float16`), but only as a storage format.
 In calculations they'll be converted to `Float32`:
```

Comment:
Okay, I will update according to your suggestion. Any suggestion on how to exemplify this behavior? The existing example was originally meant to illustrate that the result of the multiplication was a `Float32`